user-console/src/main/java/org/pentaho/mantle/client/admin/EmailAdminPanelController.java

```diff
@@ -233,30 +329,50 @@ public void onError( Request request, Throwable exception ) {
 
         public void onResponseReceived( Request request, Response response ) {
           emailConfig = JsEmailConfiguration.parseJsonString( response.getText() );
-          authenticationCheckBox.setValue( Boolean.parseBoolean( emailConfig.isAuthenticate() + "" ) );
+          String protocol = emailConfig.getSmtpProtocol();
+          selectProtocol( protocol );
           authenticationPanel.setVisible( Boolean.parseBoolean( emailConfig.isAuthenticate() + "" ) );
+          smtpPanel.setVisible( protocol.contains( Messages.getString( "smtp" ) ) );
```

Comment:
Is there a real need to use `contains` instead of `equals`? How many SMTP protocols are there? If there are several, then please create a method in a central location, such as `isSmtpProtocolFamily( protocol )`.